docs/pages/components/checkbox.mdx

```diff
@@ -21,6 +21,7 @@ import {
 function() {
   const [checkbox, setCheckbox] = React.useState(false)
   const [checkboxChecked, setCheckboxChecked] = React.useState(true)
+  const [checked, setChecked] = React.useState(false)
```

Comment:
You must be more specific about the naming here:
`[checkboxIndeterminate, setCheckboxIndeterminate]`